Release 2021-10-02

LoneRifle opened the pull request:
This release contains the following:

- #443 
- #445 
- #446 
- #448 
- #449 
- #450 
- #451 
- #453

LoneRifle committed:
feat(rte): enforce alt text for images (#443)

LinHuiqing committed:
fix(header): set header logo link as routerlink (#445)

- set chakra Link component as Link from react-router-dom
- fixes link and makes it crawlable

fix #444

LinHuiqing committed:
refactor(tags): migrate to chakra (#446)

- migrate TagBadge to chakra
- change relevant colors in theme to match with figma design

LoneRifle committed:
feat(search-results): add CitizenRequest at footer (#448)

If the user searches and finds nothing of interest, provide
CitizenRequest to prompt user to make an enquiry

- Organise imports, bring in AgencyService, CitizenRequest
- Remove min height in spacer since CitizenRequest has height
- Add CitizenRequest to SearchResult

LinHuiqing committed:
feat(logo-link): add agency main page link to agency logo (#449)

LinHuiqing committed:
fix(citizen-req): update citizen request copy (#450)

LoneRifle committed:
refactor(search): rework colors, layout (#451)

- Use gray.800 for emphasis where possible
- Declutter the page where possible, add suggestion text

LinHuiqing reviewed (commented):
Give me some time, I realised that the agency logo is weird now because I added the link oops.

LinHuiqing committed:
fix(agency-logo): set height and width of AgencyLogo Link component to 100% (#453)

- set height and width of Link to 100% so that the borders will still be flushed to a square

LinHuiqing reviewed (approved):
lgtm
